src/main/java/wolox/training/pojos/dtos/BookDTO.java

```diff
@@ -0,0 +1,41 @@
+package wolox.training.pojos.dtos;
+
+import java.text.ParseException;
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.Setter;
+import org.modelmapper.ModelMapper;
+import wolox.training.models.Book;
+
+@Setter
+@Getter
+@AllArgsConstructor
+public class BookDTO {
```

Comment:
Add image attributes (small, medium and large).

Reply:
I have just one image field on my model, Do I change the model to save this three different sizes of image from open library service?